dev/docker.sh

```diff
@@ -72,14 +37,77 @@ if [[ -z $DOCKER_PASSWORD ]]; then
 	exit 1
 fi
 
+VERSION="$TRAVIS_TAG"
+VERSION_SHORT=$( echo $VERSION | cut -d. -f1,2 )
```

Comment:
how about default "latest" ? when you build the image in your local dev env

Reply:
fixed the mismerge